[ClientBinaryManager] False error if no internet connection; can't open Mist

If no internet connection is available Mist won't be able to start.
Mist should still be able to launch without being online.

The splash-screen will show display this error strings:
`Error running downloaded binary.` and `Ethereum node connection error :'(`

While the console shows:
```javascript
[...]
[2016-12-19 17:04:34.636] [INFO] ClientBinaryManager - Initializing...
[2016-12-19 17:04:34.637] [INFO] ClientBinaryManager - Resolving path to Eth client binary ...
[2016-12-19 17:04:34.637] [INFO] ClientBinaryManager - Eth client binary path: /Users/ich/github/mist/nodes/eth/mac-x64/eth
[2016-12-19 17:04:34.637] [INFO] ClientBinaryManager - Checking for new client binaries config from: https://raw.githubusercontent.com/ethereum/mist/master/clientBinaries.json
[2016-12-19 17:04:34.641] [WARN] ClientBinaryManager - Error fetching client binaries config from repo { RequestError: getaddrinfo ENOTFOUND raw.githubusercontent.com raw.githubusercontent.com:443
    at ClientRequest.req.once.err (/Users/ich/github/mist/node_modules/got/index.js:81:21)
    at ClientRequest.g (events.js:286:16)
    at emitOne (events.js:101:20)
    at ClientRequest.emit (events.js:188:7)
    at TLSSocket.socketErrorListener (_http_client.js:308:9)
    at emitOne (events.js:96:13)
    at TLSSocket.emit (events.js:188:7)
    at connectErrorNT (net.js:1016:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  code: 'ENOTFOUND',
  message: 'getaddrinfo ENOTFOUND raw.githubusercontent.com raw.githubusercontent.com:443',
  host: 'raw.githubusercontent.com',
  hostname: 'raw.githubusercontent.com',
  method: 'GET',
  path: '/ethereum/mist/master/clientBinaries.json' }
[2016-12-19 17:04:34.642] [ERROR] ClientBinaryManager - TypeError: Cannot read property 'clients' of undefined
    at got.then.catch.then.err (/Users/ich/github/mist/modules/clientBinaryManager.js:78:45)
    at process._tickCallback (internal/process/next_tick.js:103:7)
[2016-12-19 17:04:34.643] [INFO] Sockets/node-ipc - Connect to {"path":"/Users/ich/Library/Ethereum/geth.ipc"}
[2016-12-19 17:04:34.646] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2016-12-19 17:04:35.652] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2016-12-19 17:04:36.656] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2016-12-19 17:04:37.648] [ERROR] Sockets/node-ipc - Connection failed (3000ms elapsed)
[2016-12-19 17:04:37.649] [WARN] EthereumNode - Failed to connect to node. Maybe it's not running so let's start our own...
[2016-12-19 17:04:37.649] [INFO] EthereumNode - Node type: geth
[2016-12-19 17:04:37.649] [INFO] EthereumNode - Network: main
[2016-12-19 17:04:37.650] [INFO] EthereumNode - Start node: geth main
[2016-12-19 17:04:37.651] [ERROR] EthereumNode - Failed to start node Error: Node "geth" binPath is not available.
    at EthereumNode.__startNode (/Users/ich/github/mist/modules/ethereumNode.js:324:19)
    at stop.then (/Users/ich/github/mist/modules/ethereumNode.js:257:29)
    at tryCatcher (/Users/ich/github/mist/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/ich/github/mist/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise._settlePromise (/Users/ich/github/mist/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/Users/ich/github/mist/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/Users/ich/github/mist/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/Users/ich/github/mist/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/Users/ich/github/mist/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues (/Users/ich/github/mist/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:574:20)
    at tryOnImmediate (timers.js:554:5)
    at processImmediate [as _immediateCallback] (timers.js:533:5)
[2016-12-19 17:04:37.652] [ERROR] main - Error starting up node and/or syncing Error: Node "geth" binPath is not available.
    at EthereumNode.__startNode (/Users/ich/github/mist/modules/ethereumNode.js:324:19)
    at stop.then (/Users/ich/github/mist/modules/ethereumNode.js:257:29)
    at tryCatcher (/Users/ich/github/mist/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/ich/github/mist/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise._settlePromise (/Users/ich/github/mist/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/Users/ich/github/mist/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/Users/ich/github/mist/node_modules/bluebird/js/release/promise.js:691:18)
    at Async._drainQueue (/Users/ich/github/mist/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/Users/ich/github/mist/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues (/Users/ich/github/mist/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:574:20)
    at tryOnImmediate (timers.js:554:5)
    at processImmediate [as _immediateCallback] (timers.js:533:5)
```

--- a/modules/clientBinaryManager.js
+++ b/modules/clientBinaryManager.js
@@ -73,6 +73,8 @@ class Manager extends EventEmitter {
             log.warn('Error fetching client binaries config from repo', err);
         })
         .then((latestConfig) => {
+            if(!latestConfig) return;
+
             let localConfig;
             let skipedVersion;
             const nodeVersion = latestConfig.clients[nodeType].version;

